### PR TITLE
importing multiclient's account(s)

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: roles/import-validator-keys-multiclient/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  loop:
+    - git
+    - build-essential
+    - gcc
+    - golang
+
+- name: Install ethdo
+  shell: GO111MODULE=on go get github.com/wealdtech/ethdo
+
+- name: Add ethdo to PATH
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply roles/import-validator-keys-multiclient/tasks/main.yaml]: task - "Add ethdo to PATH" is removed

File: roles/import-validator-keys-multiclient/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  blockinfile:
+    path: "~/.profile"
+    block: |
+      if [ -d "$HOME/go/bin" ] ; then
+        PATH="$PATH:$HOME/go/bin"
+      fi
+
+- name: create ethdo wallet
+  shell: ~/go/bin/ethdo wallet create --type=hd --mnemonic="{{validator_mnemonic}}" --wallet="Wallet" --wallet-passphrase="StereumP@ssw0rd"
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply roles/import-validator-keys-multiclient/tasks/main.yaml]: wallet passphrase is generated automatically

File: roles/import-validator-keys-multiclient/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+- name: create account(s)
+  shell: |
+    i=0
+    while [[ "$i" < "{{ validator_number }}" ]]
+    do
+    ~/go/bin/ethdo account create --account="Wallet/Account-$i" --path=m/12381/3600/$i/0/0 --wallet-passphrase="StereumP@ssw0rd" --passphrase="{{validator_password}}"  --allow-weak-passphrases
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply roles/import-validator-keys-multiclient/tasks/main.yaml]: now, wallet and account will use same passphrase(auto generated)

File: roles/import-validator-keys-multiclient/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- name: Put the account password into the Dirk passphrases
+  ansible.builtin.copy:
+    content: '{{ validator_password }}'
+    dest: "{{ e2dc_install_path }}/config/dirk/passphrases/account-passphrase.txt"
+  become: yes
+  become_user: "{{ stereum_user }}"
+
+- name: Put the wallet password into the Dirk passphrases
+  ansible.builtin.copy:
+    content: 'StereumP@ssw0rd'
```
Comment (stefa2k) [inline roles/import-validator-keys-multiclient/tasks/main.yaml:50]: Should use `{{ validator_password }}` I think

**Review by stefa2k — CHANGES_REQUESTED** [review]

**Review by stefa2k — APPROVED** [review]
lgtm

**Pull request merged**